bin/improver-generate-topography-bands-mask

```diff
@@ -48,12 +48,12 @@ iris.FUTURE.netcdf_promote = True
 #           within the band are set to 1.
 #   Sea points are masked and set to the default fill_value
 
```

Comment:
Please update comment to reflect new dictionary contents and behaviour.

Reply:
✅